dogpile/cache/backends/redis.py

```diff
@@ -90,6 +90,12 @@ class RedisBackend(CacheBackend):
 
      .. versionadded:: 0.5.4
 
+    :param thread_local_lock: bool, whether a thread-local Redis lock object
+     should be used. This is the default, but is not compatible with
+     asynchronous runners, as they run in a different thread than the one
+     used to create the lock.
+
+     .. versionadded:: 0.9.2
```

Comment:
I'm not sure if this line makes sense to have or if it's auto generated.

Reply:
no that's correct